src/test/kotlin/jces1209/Chromium77.kt

```diff
@@ -0,0 +1,10 @@
+package jces1209
+
+import com.atlassian.performance.tools.infrastructure.api.browser.Browser
+import com.atlassian.performance.tools.infrastructure.api.browser.chromium.CustomChromium
+import java.net.URI
+
+class Chromium77 : Browser  by CustomChromium(
+    chromiumUri = URI("https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F681090%2Fchrome-linux.zip?generation=1564102126574765&alt=media"),
```

Comment:
You found it! 😄 
For posterity, there's a [guide how to find the proper Chrome build number](https://www.chromium.org/getting-involved/download-chromium).

Reply:
Yup, i'm not really sure why ChromeDriver from googleapis.com does not work (from de place i got chromium browser) but the one from:
https://chromedriver.chromium.org/
works, maybe i'm not understanding it correctly though